src/mongoose/findGlobal.ts

```diff
@@ -19,8 +17,8 @@ export async function findGlobal<T extends TypeWithID = any>(
   let doc = await Model.findOne(query).lean() as any;
 
   if (!doc) {
-    doc = {};
-  } else if (doc._id) {
+    return null;
```

Comment:
this seems like it might be problematic, unless we manage this internally within the global find operation itself. If the global has not yet been created, Payload should send back an empty object, not null. Something to check on

Reply:
> this seems like it might be problematic, unless we manage this internally within the global find operation itself.

We do actually. See [this line](https://github.com/payloadcms/payload/pull/2918/files#diff-5dd837eec1739318f1361e26ddce331044d486a1ee7b0fccebf91caeb0d288b0R53).

If we return {} here, it would break this: https://github.com/payloadcms/payload/pull/2918/files#diff-02c9db90817e70060d52ef9b7c46d645d82c83012270b3bb8eea40169628e6ddL30 as Boolean({}) seems to resolve to `true`.

So either we leave it like that, or change it back to return {} but then modify the code in `getLatestGlobalVersion` which checks if the global exists. What do you think?

Reply:
OK I'm good with what you have then!